src/layout/sidebar/SidebarColumns.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from "react";
```

Comment:
I'm thinking of moving this to the Alethio/explorer-ui package instead, possibly together with the Sidebar component